src/components/aria-live/aria-live.spec.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  inject,
+  TestComponentBuilder, ComponentFixture
+} from 'angular2/testing';
+import {
+  it,
+  describe,
+  expect,
+  beforeEach,
+} from '../../core/facade/testing';
+import {Component} from 'angular2/core';
+import {By} from 'angular2/platform/browser';
+import {MdAriaLive} from './aria-live';
+
+export function main() {
+  describe('MdAriaLive', () => {
+    let builder: TestComponentBuilder;
+    let timeoutMock: any;
+
+    beforeEach(inject([TestComponentBuilder], (tcb: TestComponentBuilder) => {
+      builder = tcb;
+    }));
+
+    beforeAll(() => {
+      timeoutMock = mockTimeout();
```

Comment:
You should be able to do this more easily with Angular 2's `fakeAsync`; check out the portal tests for an example.